client/packages/system/src/Patient/ProgramEnrolment/Components/DetailModal.tsx

```diff
@@ -6,10 +6,10 @@ import {
   useDialog,
   useJsonForms,
 } from '@openmsupply-client/common';
-import { useProgramEnrolment } from './api/hooks';
-import { usePatientModalStore } from '../hooks';
-import { PatientModal } from '../PatientView';
-import { usePatient } from '../api';
+import { useProgramEnrolment } from '../api/hooks';
```

Comment:
ah yeah. sorry. I had the `ProgramSearchModal` in here at one point, so I moved the components into `Components`. Then realised that the api is patient specific in here, so moved the search modal out.